### ISSUE TITLE
azurerm_application_insights could not target application_type = "MobileCenter"

### ISSUE THREAD

**Issue by guitarrapc** [terraform-providers/terraform-provider-azurerm#1815]
hi, I'm trying import existing application insight resource. I faced to when Application Insight is made for MobileCenter, application_type would not match with any value.

## Step to reproduce

1. Create Application insight from AppCenter.
2. Import resource with Terraform.
3. run `terraform plan` will get mis-match for "application_type".

## terraform configuration

```
resource "azurerm_application_insights" "AppCenter" {
  name                = "AppCenter"
  location            = "westus2"
  resource_group_name = "resourceGroupName"
  application_type    = "other"
}
```

## Actual result

```
application_type:    "MobileCenter" => "Other" (forces new resource)
```

## Expected

If I can set `MobileCenter` as a application_type, issue will not happen.

```
resource "azurerm_application_insights" "AppCenter" {
  name                = "AppCenter"
  location            = "westus2"
  resource_group_name = "resourceGroupName"
  application_type    = "MobileCenter"
}
```

terraform plan will not detect change.

```
no diff
```

## Question

Am I missing some feature to match application_type?

**Comment by tombuildsstuff** [comment]
hey @guitarrapc 

Just to let you know that this has been released in v1.14.0 of the AzureRM Provider which is now available: https://github.com/terraform-providers/terraform-provider-azurerm/blob/v1.14.0/CHANGELOG.md

Thanks!

**Comment by guitarrapc** [comment]
Thank you so much!! I'll take a look and deploy to dev/staging!
Thanks!
